cli/flox/src/utils/openers.rs

```diff
@@ -173,8 +170,59 @@ where
 
 #[cfg(test)]
 mod tests {
+
     use super::*;
 
+    /// On Linux, Browser::detect() finds xdg-open if it's in path
+    ///
+    /// TODO: we might want to better simulate an actual display and opener
+    #[test]
+    #[cfg(target_os = "linux")]
+    fn test_browser_detect_finds_opener_in_path() {
+        use std::fs::File;
+
+        use tempfile::tempdir;
+
+        let temp_dir = tempdir().unwrap();
+        let xdg_open = temp_dir.path().join("xdg-open");
+        File::create(&xdg_open).unwrap();
+
+        temp_env::with_vars(
+            [
+                ("SSH_TTY", None),
+                ("DISPLAY", Some("1")),
```

Comment:
suggestion: test with `WAYLAND_DISPLAY` as well

Reply:
WAYLAND_DISPLAY was not tested in the integration tests, so while that would probably be good I think it's out of scope

Reply:
I think we'll not really get back to doing that anytime soon if we don't do it here.
OTOH it's a rather trivial test given that it tests the same codepath as `DISPLAY`, so  lets not block on it.

Reply:
Agreed, it's just there's 22 more expect tests to go so I'm trying to keep scope down 😂